src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -583,6 +583,10 @@ export default {
 		height: var(--icon-size);
 	}
 
+	&__input:focus-visible + label {
+		outline: 2px solid var(--color-main-text) !important;
```

Comment:
Is `outline` the standard for showing `focus-visible`? It creates issues when multiple elements are close together:
![checkbox](https://github.com/nextcloud/nextcloud-vue/assets/2496460/5aaab2e0-535f-4bd3-b053-a598bfd45f04)

Would it be an option to use `border` instead? When we give the `label` a `transparent` border by default, reduce the padding by `2px` in all directions and switch to a colored border when `focus-visible`, it creates a similar effect, but without the overlapping:

![checkbox_border](https://github.com/nextcloud/nextcloud-vue/assets/2496460/ac29df31-6eb8-4e7d-990c-5c374751e1c3)

```css
.checkbox-radio-switch__input:focus-visible + label[data-v-1b902817] {
  /* outline: 2px solid var(--color-main-text) !important; */
  border: 2px solid var(--color-main-text) !important;
}

.checkbox-radio-switch__label[data-v-1b902817] {
  /* padding: 4px 14px; */
  padding: 2px 12px;
  border: 2px solid transparent;
}
```

Reply:
@szaimen I guess you are suggesting we start using `--color-primary-element` in this, right?

Reply:
Yes 👍

Reply:
@szaimen done!